Backpack/src/main/java/net/skyscanner/backpack/flare/BpkFlare.kt

```diff
@@ -128,12 +146,26 @@ open class BpkFlare @JvmOverloads constructor(
       throw IllegalStateException("BpkFlare should have only one child")
     }
 
-    if (insetPaddingMode == InsetPaddingMode.BOTTOM) {
-      val paddingBottom = child.paddingBottom + pointerMask.height
-      if (child.paddingStart > 0 || child.paddingEnd > 0) {
-        child.setPaddingRelative(child.paddingStart, child.paddingTop, child.paddingEnd, paddingBottom)
-      } else {
-        child.setPadding(child.paddingLeft, child.paddingTop, child.paddingRight, paddingBottom)
+    when (insetPaddingMode) {
+      InsetPaddingMode.NONE -> {}
+      InsetPaddingMode.BOTTOM, InsetPaddingMode.TOP -> {
```

Comment:
Can we do this in the 2 different branches and use some sort of function/extension to handle this?

Reply:
Done